cmd/common.go

```diff
@@ -16,6 +16,8 @@ import (
 
 const maxMFATokenPromptAttempts = 5
 
+var globalMFACodeMap = make(map[string]string)
```

Comment:
I could use some education here! I don't know very much Go at all.

This was pretty easy to read and works totally fine. To check my understanding, it looks like we're setting up a map here where the key and value are both going to be the same MFA code. What are the advantages of using a map here instead of some kind of list or array (or I guess a slice in Go) and then checking for the presence of the MFA code in that? I'd like to learn the Go idioms here!

Reply:
I also do not know very much Go at all too.

My intention was to do the simplest thing. Apparently arrays and slices do not have a generic method for the "contains" function. Extra defensive code would have to be implemented to check for uniqueness of an element for arrays and slices. If Go had Sets I would've used that, but it doesn't so I used a Map.

Reply:
I ran into some stuff about how to implement contains for native collection types and I'm like wow I'd really rather not have to do that. Using a map makes sense to me, thanks for explaining!